OpenSim/Tools/Test/testVisualization.cpp

```diff
@@ -476,7 +476,7 @@ void populate_wrapModelPrimitives(SimTK::Array_<DecorativeGeometry>& stdPrimitiv
     // that DecorativeCylinder is Y aligned while WrapCylinder is Z aligned
     cylTransform.updR().setRotationFromAngleAboutX(SimTK_PI / 2);
     stdPrimitives.push_back(
-        DecorativeCylinder(.025, .05).setBodyId(0).setColor(SimTK::Cyan)
+        DecorativeCylinder(.025, .05).setBodyId(0).setColor(Vec3(0, 0.2, 0.8))
```

Comment:
We used Cyan in the past so that testers know what to expect if they turn visualization on, is there a reason to revert to the RGB values?